### PR TITLE
Javascript form validator

### PR DESCRIPTION
**Pull request by StanimirIliev**
link -> https://stanimiriliev.github.io/validator/

### CODE REVIEW

File: validator/validator.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+function validate() {
```
Comment (mgenov) [inline validator/validator.js]: This looks too concrete. Adding of a new field will force you to modify this function which doesn't look right.

Think for a better way for doing of the validation by using regular expressions.

Comment (StanimirIliev) [reply validator/validator.js]: What about now?

Comment (mgenov) [reply validator/validator.js]: If you take a close look at the code below you will see that: document.getElementById("field").value is duplicated on many places. 

The only difference between these fields is the error message.

```javascript
val validators = [{
  {name: "egn", validator: new RegexValidator(/[0-9]{10}/), "Invalid EGN"},
  {name: "email", validator: new RegexValidator(/* email validation regex*/), "Invalid Email"},
}]

var errors = validate(validators) 
if (errors.length > 0) {
 // show errors 
}
```

**Comment by mgenov** [comment]
Why you didn't say that PR is ready  ?

**Pull request merged**